invenio_iiif/__init__.py

```diff
@@ -6,90 +6,89 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Image cropping and previewer API.
+"""Invenio module to serve images complying with IIIF standard.
 
 Invenio-IIIF integrates
-`Invenio-Records-Files <https://invenio-records-files.readthedocs.io>`_ with
-`Flask-IIIF <https://flask-iiif.readthedocs.io/en/latest/>`_
-to provide an endpoint for serving images
-with the International Image Interoperability Framework (IIIF) API standards.
-To be a bit more specific Invenio-IIIF is registering the provided endpoints
-from Flask-IIIF and it is using the files permissions provided
-by Invenio-Files-Rest.
-On top of that it is providing additional functionalities based on the IIIF
-standards. For more info check here (https://iiif.io/)
+`Invenio-Records-Files <https://invenio-records-files.rtfd.io>`_ with
+`Flask-IIIF <https://flask-iiif.rtfd.io>`_ to provide an endpoint for serving
+images complying with the `International Image Interoperability Framework
+(IIIF) <https://iiif.io/>`_ API standards.
+
+Invenio-IIIF registers the REST API endpoint provided by Flask-IIIF in the
+Invenio instance through entry points. On each request, it delegates
+authorization check to Invenio-Files-REST, load the file from the Invenio
+bucket and serve it after Flask-IIIF manipulation.
 
 Initialization
 --------------
-When this package is installed with Flask-IIIF it will register the second's
-blueprints under the configurable prefix in config['IIIF_API_PREFIX']
+You can define the IIIF API prefix by setting the Invenio configuration
+``IIIF_API_PREFIX``.
 
 Usage
 -----
-*Prerequisites*: Having an image or pdf stored under an Invenio File Object
-instance.
-
->>> from invenio_iiif.utils import ui_iiif_image_url
+*Prerequisites*: having an image or pdf stored as an Invenio File ObjectVersion
+instance in a Bucket.
 
->>> from invenio_files_rest.models import ObjectVersion
+While Flask-IIIF requires in the path of the URL the UUID of the image to
```

Comment:
I think it is hard to understand this sentence. Could we simplify it a bit? `While Flask-IIIF requires the UUID in the URL path of the retrievable image`

Reply:
I think the current version is a bit clearer in this case :)